### PR TITLE
[docs] fix typos in page_transitions_theme.dart and tooltip.dart

### LINKED ISSUE #106263: [docs] word repeated in page_transitions_theme.dart docs
**Screenshot**

Screenshot

https://api.flutter.dev/flutter/material/PageTransitionsTheme-class.html

### PR DESCRIPTION
**Pull request by albinpk**
- Remove a repeated word in packages/flutter/lib/src/material/page_transitions_theme.dart
- Fix typo in packages/flutter/lib/src/material/tooltip.dart

**Fix #106263** 

## Pre-launch Checklist

- [X] I read the [Contributor Guide] and followed the process outlined there for submitting PRs.
- [X] I read the [Tree Hygiene] wiki page, which explains my responsibilities.
- [X] I read and followed the [Flutter Style Guide], including [Features we expect every widget to implement].
- [x] I signed the [CLA].
- [X] I listed at least one issue that this PR fixes in the description above.
- [ ] I updated/added relevant documentation (doc comments with `///`).
- [ ] I added new tests to check the change I am making, or this PR is [test-exempt].
- [ ] All existing and new tests are passing.

### CODE REVIEW

**Review by abinvs-2019 — COMMENTED** [review]
Fixed typos in page page_transitions_theme.dart and tooltip.dart.

**Review by TahaTesser — APPROVED** [review]

**Comment by TahaTesser** [comment]
Before this can be merged this need a test exemption and second review from @Hixie

**Review by Piinks — APPROVED** [review]
Flutter_LGTM
Thanks!

**Pull request merged**
